src/3rdparty/squirrel/squirrel/sqmem.cpp

```diff
@@ -9,8 +9,11 @@
 #include "../../../core/alloc_func.hpp"
 #include "../../../safeguards.h"
 
-void *sq_vm_malloc(SQUnsignedInteger size){	return MallocT<char>((size_t)size); }
 
-void *sq_vm_realloc(void *p, SQUnsignedInteger oldsize, SQUnsignedInteger size){ return ReallocT<char>(static_cast<char*>(p), (size_t)size); }
+#ifdef SQUIRREL_DEFAULT_ALLOCATOR
+void *sq_vm_malloc(SQUnsignedInteger size) { return return MallocT<char>((size_t)size); }
```

Comment:
`return return` and all the whitespaces changes adding noise in the diff (I know it can be hidden)

Reply:
I'm not sure if I can avoid fixing the whitespace, if I want to keep the commit checker happy. The original file has tabs in the middle of the line.

Reply:
Yeah, it's probably like when preprocessor stuff with the old style appears in a diff.